docs/operations.md

```diff
@@ -0,0 +1,143 @@
+# Pipelines-Service operations
+
+## Introduction
+
+The aim of this document is to introduce various considerations, processes and tasks required for operating Pipelines-Service.
+The primary audience are teams willing to deploy and operate Pipelines-Service.
+The scope is focused on the operational aspects. This document does not go into the details of the implementation as far as it is not required from an operational point of view.
+
+## Components
+
+**Phase 1**
+
+![Components](./images/components.png)
+
+### Management
+
+The components of the management layer are pre-requisite for the automation and the deployment of the Pipelines-Service platform.
+
+#### GitHub repository
+
+This is the repository where this file and the components for automating the deployment and operation of Pipelines-Service are made available and maintained.
+
+#### GitOps repository
+
+The GitOps repository contains the files providing the intended state and configuration of a Pipelines-Service installation.
+
+A separate document will be created to present the directory structure and the possible approaches for customization.
+
+#### Image repository
+
+Pipelines as Code uses PipelineRuns for automation. An online or offline repository needs to serve the images for the Tasks of the PipelineRuns.
+
+#### Argo CD
+
+Pipelines-Service takes a GitOps approach and leverages Argo CD for the purpose. With phase 1, an in-cluster management approach is taken. A separate instance of Argo CD runs on every compute cluster and the interaction with Argo CD is done through the kubernetes API of the cluster.
+
+### Infrastructure
+
+The infrastructure components are not directly managed by the Pipelines-Service platform, which however relies on them. They may get managed by the same or a different team depending on the company size and organization.
+
+#### kcp
+
+kcp allows decoupling the control plane and compute. It brings better multi-tenancy and greater scalability.
+End users interact with Pipelines-Service through their kcp workspace or a layered product on top of it. They don't need to interact with any of the lower layer components.
+
+#### Compute clusters
+
+Computer clusters are traditional Kubernetes clusters. Once registered to kcp they bring the compute power necessary for processing the PipelineRuns.
```

Comment:
We have a few shortcuts where we leverage OLM for operator installation and don't offer (yet) equivalents based on pure manifests but all what we do should be possible with kubernetes distributions other than OpenShift.

Reply:
Throughout our repo, we are currently focusing on OpenShift clusters and them alone. Although it is possible to tweak stuff and make it possible to run on other clusters, for now, it is not available. It would be a bit conflicting if we include Kubernetes clusters here but mention OpenShift clusters everywhere else.

Few eg: 
https://github.com/openshift-pipelines/pipelines-service/tree/main/ckcp#kcp-in-openshift - we had this changed from kcp-in-kubernetes to kcp-in-openshift
https://github.com/openshift-pipelines/pipelines-service/tree/main/gitops#prerequisites - Everything listed under this section is OpenShift specific.

Reply:
I tried to better present the current state of things. Let me know whether it matches what you have in mind.